src/main/java/pl/coderslab/service/OrderService.java

```diff
@@ -0,0 +1,135 @@
+package pl.coderslab.service;
+
+import org.springframework.stereotype.Service;
+import pl.coderslab.dto.OrderDtoNew;
+import pl.coderslab.dtoread.OrderDtoReadNew;
+import pl.coderslab.model.Order;
+import pl.coderslab.repository.OrderRepository;
+
+import javax.persistence.EntityExistsException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+public class OrderService implements CrudService<Order>, OrderServiceMethods {
+
+
+    private final OrderRepository orderRepository;
+    private final NewOrderNumberGenerator newOrderNumberGenerator;
+
+    public OrderService(OrderRepository orderRepository, NewOrderNumberGenerator newOrderNumberGenerator) {
+        this.orderRepository = orderRepository;
+        this.newOrderNumberGenerator = newOrderNumberGenerator;
+    }
+
+
+    @Override
+    public void add(Order order) {
+
```

Comment:
W takich sytuacjach jak jesze nie masz czegoś zaimplementowanego to warto rzucić wyjątek aby potem nie zastanawiać się dlaczego się nie dodaje :)